src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -268,6 +268,17 @@ extern void setDbPrefix(std::string _dbPrefix)
 
 /*****************************************************************************
 *
+* dbPrefixGet -
+*/
+const char* dbPrefixGet(void)
```

Comment:
I changed the return type from _const char*_ to _const std::string&_.  The compiler accepts it, so I have no reason to believe it will give any problems.

My only concern about this right now, having the following code:

    const std::string& stringFix()
    {
        return  "A long string";
    }

    std::string sss = stringFix();

is that I don't know how the compiler translates this ...
Supposedly *stringFix* returns a pointer to a std::string.
My guess: std::string has an operator '=' that accepts a pointer to a std::string.